src/content/structured/components/back-to-top/guidance.mdx

```diff
@@ -59,6 +70,17 @@ Consider using it on all apps to improve usability for everyone, including keybo
   caption="Display the back to top button when a page has been scrolled."
 />
 
+### Icon-only
+
+The icon-only back to top button should be used when a page is zoomed, to minimise overlapping content.
```

Comment:
With my current work on [this ticket](https://github.com/mi6/ic-design-system/issues/993), it would be great to have a line here to say that any icon can be used and to refer to the [page dedicated to guidance for icons](https://design.sis.gov.uk/styles/icons). Thanks ☺️

Reply:
I don't think this is needed here as the user can't slot any icon, it's only the arrow of the back to top used